### ISSUE TITLE
Unable to install new version or run old portable version

### ISSUE THREAD

**Issue by greenvas** [HearthSim/Hearthstone-Deck-Tracker#2932]
Game stopped working once, and I was never able to run or install it again. I list the two error details I get when trying to install it.
I run Windows 7 Ultimate and .NET Frameworks 4.6.3

Log Name:      Application
Source:        .NET Runtime
Date:          07/11/2016 20:02:06
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      PC
Description:
Application: Update.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 6AB9EE4C

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1026</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2016-11-07T23:02:06.000000000Z" />
    <EventRecordID>6078</EventRecordID>
    <Channel>Application</Channel>
    <Computer>PC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Application: Update.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 6AB9EE4C
</Data>
  </EventData>
</Event>

Log Name:      Application
Source:        .NET Runtime
Date:          07/11/2016 20:02:05
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      PC
Description:
Application: HearthstoneDeckTracker.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 6AB9EE4C

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1026</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2016-11-07T23:02:05.000000000Z" />
    <EventRecordID>6076</EventRecordID>
    <Channel>Application</Channel>
    <Computer>PC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Application: HearthstoneDeckTracker.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 6AB9EE4C
</Data>
  </EventData>
</Event>

**Comment by riQQ** [comment]
> Game stopped working once, and I was never able to run or install it again.

What do you mean by that? Did HDT stop working or was it Hearthstone?

**Comment by greenvas** [comment]
While playing Hearthstone the tracker crashed giving me an error message
(HDR has stopped working)

**Comment by riQQ** [comment]
Is there a log entry in the windows event log for the HDT crash itself (not the installer crash?
The installer crash is likely a Squirrel issue. Squirrel is the framework used for installing and updating.

**Comment by greenvas** [comment]
I tracked the first error I got, I assume its the one that happend when it
crashed for the first time.

Log Name:      Application
Source:        .NET Runtime
Date:          03/11/2016 23:22:06
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      PC
Description:
Application: Hearthstone Deck Tracker.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 5A1BEE4C

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1026</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2016-11-04T02:22:06.000000000Z" />
    <EventRecordID>5160</EventRecordID>
    <Channel>Application</Channel>
    <Computer>PC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Application: Hearthstone Deck Tracker.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 5A1BEE4C
</Data>
  </EventData>
</Event>

Log Name:      Application
Source:        Application Error
Date:          03/11/2016 23:22:08
Event ID:      1000
Task Category: (100)
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      PC
Description:
Faulting application name: Hearthstone Deck Tracker.exe, version: 1.1.0.2,
time stamp: 0x57e2b042
Faulting module name: AppProfiles.dll, version: 1.0.5.5, time stamp:
0x5277f418
Exception code: 0xc0000005
Fault offset: 0x0002ee4c
Faulting process id: 0x7b8
Faulting application start time: 0x01d236423c4de581
Faulting application path: D:\Imágenes
(2)\Hearthstone.Deck.Tracker-v1.0.5\Hearthstone Deck Tracker\Hearthstone
Deck Tracker.exe
Faulting module path: C:\Program Files (x86)\RadeonPro\AppProfiles.dll
Report Id: 7bc70968-a235-11e6-813b-20cf30e16d15
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" />
    <EventID Qualifiers="0">1000</EventID>
    <Level>2</Level>
    <Task>100</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2016-11-04T02:22:08.000000000Z" />
    <EventRecordID>5161</EventRecordID>
    <Channel>Application</Channel>
    <Computer>PC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Hearthstone Deck Tracker.exe</Data>
    <Data>1.1.0.2</Data>
    <Data>57e2b042</Data>
    <Data>AppProfiles.dll</Data>
    <Data>1.0.5.5</Data>
    <Data>5277f418</Data>
    <Data>c0000005</Data>
    <Data>0002ee4c</Data>
    <Data>7b8</Data>
    <Data>01d236423c4de581</Data>
    <Data>D:\Imágenes (2)\Hearthstone.Deck.Tracker-v1.0.5\Hearthstone Deck
Tracker\Hearthstone Deck Tracker.exe</Data>
    <Data>C:\Program Files (x86)\RadeonPro\AppProfiles.dll</Data>
    <Data>7bc70968-a235-11e6-813b-20cf30e16d15</Data>
  </EventData>
</Event>

2016-11-08 15:27 GMT-03:00 riQQ <notifications@github.com>:
